src/components/asset/AssetBorrowInfo.tsx

```diff
@@ -125,6 +127,26 @@ export const AssetBorrowInfo: React.FC<ReserveComponentProps> = ({ poolId, asset
           </Typography>
         </Box>
       </Row>
+      <LinkBox
+        sx={{ width: '100%' }}
+        to={{ pathname: '/borrow', query: { poolId: poolId, assetId: reserve.assetId } }}
+      >
+        <Row>
+          <OpaqueButton
+            palette={theme.palette.borrow}
+            sx={{
+              width: '100%',
+              alignItems: 'center',
+              justifyContent: 'center',
+
+              margin: '6px',
+              fontSize: '24px',
+            }}
+          >
+            Borrow
+          </OpaqueButton>
```

Comment:
the text here is too large, I prefer if it was the default.

we should also indicate that this will cause the router to do something. IMO we should copy the `Details ->` button on the asset action pages.